benchmark_runner/workloads/workloads_operations.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         self._workload = self._environment_variables_dict.get('workload', '')
         self._build_version = self._environment_variables_dict.get('build_version', '')
         self._workloads_odf_pvc = list(self._environment_variables_dict.get('workloads_odf_pvc', ''))
-        self._odf_pvc = self._environment_variables_dict.get('odf_pvc', '')
+        self._odf_pvc = self._environment_variables_dict.get('odf_pvc', True)
```

Comment:
By default is True?

Reply:
@ebattat I believe this is the current default?

Reply:
Yes, True in the current default